utils/validate_pr_quickstarts.js

```diff
@@ -0,0 +1,357 @@
+'use strict';
+
+const glob = require('glob');
+
+const { fetchPaginatedGHResults } = require('./github-api-helpers');
+const {
+  findMainQuickstartConfigFiles,
+  readQuickstartFile,
+  removeRepoPathPrefix,
+} = require('./helpers');
+const {
+  fetchNRGraphqlResults,
+  translateMutationErrors,
+} = require('./nr-graphql-helpers');
+
+const GITHUB_API_URL = process.argv[2];
```

Comment:
Realized we didn't use the `pathsToSanitize` helper. I think we should use a helper here, and I'll propose a change to a function name of `passedProcessArguments` so it could be accessed as `passedProcessArguments[0]`. 

Think the `sanitized` naming is from a different context here: https://github.com/tyreer/newrelic-quickstarts/blob/main/utils/sanitize_dashboards.js#L4